aten/src/ATen/native/TensorShape.cpp

```diff
@@ -42,6 +43,18 @@ Tensor& set_(Tensor& result, Storage source) {
   return result.set_(source, 0, static_cast<int64_t>(source.size()), {});
 }
 
+static auto registry_set_ = torch::RegisterOperators()
+  .op(torch::RegisterOperators::options()
+    .schema("aten::set_.source_Storage(Tensor(a!) self, Storage source) -> Tensor(a!)")
+    .impl_unboxedOnlyKernel<Tensor&(Tensor&, Storage), &set_>(DispatchKey::CPUTensorId))
+  ;
```

Comment:
Note that the syntax will change when the new API shows up.

Reply:
We spoke and ezyang suggested that I continue with the old API since he will have to convert it all in bulk anyway.